control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -72,6 +73,16 @@ pub struct VolumeState {
     pub usage: Option<VolumeUsage>,
 }
 
+/// Volume properties.
+#[derive(
+    Serialize, Deserialize, EnumString, Debug, EnumCountMacro, EnumIter, PartialEq, Clone, Copy,
```

Comment:
Let's not do implicit Copy here, new attr size could be larger as we add more things 
Also do we need all of the strums derives btw?

Reply:
strum is removed. But I think `copy` is needed while passing this enum in `set_property` function.

Reply:
Let's remove Copy, just use clone whenever needed